### DISCUSSION TITLE
POSSIBILITA' DI METTERE IL TEMA DEL REG. SCURO vi scongiuro

### DISCUSSION THREAD

**Discussion by filipguerra** [iisgiua/giuaschool#795] [Idee e proposte]
sarebbe molto innovativo anche il cambiamento per quanto riguarda il tema del registro elettronico (chiaro, scuro, ecc)

**Comment by trinko** [comment]
Puoi sempre modificare autonomamente i template con gli stili che preferisci, usando i CSS.

**Comment by filipguerra** [comment]
da dove, ti prego

Istituto Superiore BG_IS


Il giorno sab 14 dic 2024 alle 20:16 Antonello Dessì <
***@***.***> ha scritto:

**Comment by filipguerra** [comment]
non sono per niente capace, non sono capace con i codici né niente ,
riusciresti a indicarmelo per favore 🙏

Istituto Superiore BG_IS


Il giorno sab 14 dic 2024 alle 20:17 Filip Guerra ***@***.***>
ha scritto:
